sklearn/impute.py

```diff
@@ -98,28 +92,23 @@ class SimpleImputer(BaseEstimator, TransformerMixin):
         a new copy will always be made, even if `copy=False`:
 
         - If X is not an array of floating values;
-        - If X is sparse and `missing_values=0`;
-        - If `axis=0` and X is encoded as a CSR matrix;
-        - If `axis=1` and X is encoded as a CSC matrix.
+        - If X is sparse and `missing_values=0`.
 
     Attributes
     ----------
     statistics_ : array of shape (n_features,)
-        The imputation fill value for each feature if axis == 0.
+        The imputation fill value for each feature.
 
     Notes
     -----
-    - When ``axis=0``, columns which only contained missing values at `fit`
-      are discarded upon `transform`.
-    - When ``axis=1``, an exception is raised if there are rows for which it is
-      not possible to fill in the missing values (e.g., because they only
-      contain missing values).
+    - The sklearn.preprocessing.Imputer has been renamed to 
```

Comment:
I'm not sure this is helpful. People reading here have already found SimpleImputer. It might be helpful to just say that to do imputation across rows, they can use...